### PR TITLE
Disable a few tests in preparation to enabling PE+TE

### PR DESCRIPTION
**Pull request by Krovatkin**
Disable a few tests in preparation to enabling PE+TE
Next PR: https://github.com/pytorch/pytorch/pull/45396

### CODE REVIEW

File: test/test_jit_cuda_fuser.py [hunk 1]
```diff
@@ -513,7 +513,7 @@ def t(x: torch.Tensor, y: torch.Tensor, z: float):
         self.assertEqual(o, jit_o)
         self.assertGraphContains(t_jit.graph_for(x, y, 2.0), FUSION_GROUP)
 
-    @unittest.skipIf(not RUN_CUDA, "requires CUDA")
+    @unittest.skip("temp disable while switching to Profiling Executor")
```
Comment (ZolotukhinM) [inline test/test_jit_cuda_fuser.py:516]: Nit: we probably want to still keep `@unittest.skipIf(not RUN_CUDA, "requires CUDA")`. AFAIK, these decorators can be combined.

File: test/test_jit_cuda_fuser.py [hunk 2]
```diff
@@ -614,10 +614,7 @@ def test_reduction(self):
                     perm1 = range(len(x))
                     self._reduction_helper(x, axes, torch.float32, "cuda", perm0, perm1)
 
-    @unittest.skipIf(not RUN_CUDA, "requires CUDA")
-    @unittest.skipIf(GRAPH_EXECUTOR != ProfilingMode.PROFILING and GRAPH_EXECUTOR !=
-                     ProfilingMode.LEGACY, "Requires fusion optimization pass to be effective")
-    @skipIfRocm
+    @unittest.skip("temp disable while switching to Profiling Executor")
```
Comment (ZolotukhinM) [inline test/test_jit_cuda_fuser.py:617]: Ditto.

**Review by ZolotukhinM — APPROVED** [review]
👍

File: test/test_jit_cuda_fuser.py (hunk 1 above)
Comment (Krovatkin) [reply test/test_jit_cuda_fuser.py]: a good point! updated the PR

**Pull request closed without merging**